docs/usage/storage/volumeclass.md

```diff
@@ -1 +1,38 @@
-# VolumeClass
\ No newline at end of file
+# VolumeClass
+The `VolumeClass` in `IronCore` is a Kubernetes-like abstraction that defines a set of parameters or configurations for provisioning storage resources through the `IronCore Runtime Interface (IRI)`. It is conceptually similar to Kubernetes `StorageClass`, enabling users to specify the desired properties for an IronCore `Volume` resource creation.
```

Comment:
The reference for k8s storageclasses is good, please add this to the bucketclass introduction as well.